scan-payment/src/main/java/com/getbouncer/scan/payment/Image.kt

```diff
@@ -125,127 +144,58 @@ fun Bitmap.fragment(
  */
 @CheckResult
 fun Bitmap.zoom(
```

Comment:
this method is starting to get very specific to our screen detect use case. perhaps we should move the logic in to the ScreenDetect class.